CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to
 as per the xAPI specification
 - Use batch/v1 api in cronjob_pipeline manifest
 - Use autoscaling/v2 in HorizontalPodAutoscaler manifest
+- Add the ability to ignore health check routes for Sentry transactions
```

Comment:
Maybe also add the following as its a breaking change... sic!
```suggestion
- Rename the `RALPH_EXECUTION_ENVIRONMENT` configuration environment variable to `RALPH_SENTRY_EXECUTION_ENVIRONMENT`
```

Reply:
I'm postponing this breaking change for a bigger v4.0.0